paddle/scripts/submit_local.sh.in

```diff
@@ -112,13 +112,13 @@ fi
 
 case "$1" in
     "train")
-        ${DEBUGGER} $MYDIR/../opt/paddle/bin/paddle_trainer ${@:2}
+        ${DEBUGGER} $MYDIR/../../opt/paddle/bin/paddle_trainer ${@:2}
```

Comment:
Thanks @typhoonzero and @tensor-tang , I think the binary file `paddle_train/paddle_pserver_main/...` is placed under two different directories in the Docker image just because we installed the deb package while building production Docker image.For now we can install paddle with `whl` instead of `deb` package, I'll update this PR and please review after a moment.